master/reference/node/configuration.md

```diff
@@ -176,10 +177,15 @@ IP6_AUTODETECTION_METHOD=interface=eth.*
 
 The `calico/node` container supports an exec readiness endpoint.
 
-Here is the help text:
+To retrieve the help:
+
+```
+docker exec calico-node /bin/readiness -h
```

Comment:
This command is actually wrong now - it should be this:

```
docker exec calico-node /bin/calico-node -h
```

Reply:
It will return more than just readiness information, though, so we should maybe clean this section up so it doesn't refer you to running that command at all, and instead just gives the supported options.

From here: https://github.com/projectcalico/node/blob/master/cmd/calico-node/main.go#L42-L45

We support:

- `-bird-ready`
- `-bird6-ready`
- `-felix-ready`

Reply:
OK take a look at the update and let me know does it look right, I wasn't quite sure what the command should be, exactly.